.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
minor: why .js

Reply:
By changing to js, ​​it can be included in the eslint formatting target 👍 

https://github.com/react-hook-form/devtools/blob/a2c540f5e88de806e729ba0b9168fc7ea93b5a7f/package.json#L19